README.md

```diff
@@ -43,14 +43,6 @@ $ sudo apt-get install openssl libssl-dev net-tools
 $ sudo apt-get install libqt5websockets5 libqt5websockets5-dev
 ```
 
-- For Ubuntu `16.04`:
```

Comment:
Can we remove the line above : "For Ubuntu 18.04+:" now that theres only one case in this section